kong/tools/grpc.lua

```diff
@@ -13,7 +13,7 @@ local grpc = {}
 
 local function safe_set_type_hook(type, dec, enc)
   if not pcall(pb.hook, type) then
-    ngx.log(ngx.NOTICE, "no type '" .. type .. "' defined")
+    ngx.log(ngx.DEBUG, "no type '" .. type .. "' defined")
```

Comment:
Log level changed.

Reply:
yes, this message appears constantly because the protobuf state is global; but in most cases can be safely ignored so i wanted to quiet it down a bit.  still don't want to remove it because it _is_ useful when adding new "special types"